yarp.gemspec

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "yarp"
+  spec.version = "0.4.0"
+  spec.authors = ["Shopify"]
+  spec.email = ["ruby@shopify.com"]
+
+  spec.summary = "Yet Another Ruby Parser"
+  spec.homepage = "https://github.com/ruby/yarp"
+  spec.license = "MIT"
+
+  spec.require_paths = ["lib"]
+  spec.files = Dir.glob([
+    "CODE_OF_CONDUCT.md",
+    "config.yml",
+    "CONTRIBUTING.md",
+    "Gemfile",
+    "LICENSE.md",
+    "Makefile",
+    "Rakefile",
+    "yarp.gemspec",
+    "bin/**/*",
```

Comment:
Should bin be in the gem. It looks like internal scripts to develop this project.